pkg/k8s/diverts/divert.go

```diff
@@ -32,13 +33,14 @@ func DivertIngress(ctx context.Context, m *model.Manifest, fromIn *networkingv1.
 		if !oktetoErrors.IsNotFound(err) {
 			return err
 		}
-		in = translateIngress(m, fromIn, "")
+		in = translateIngress(m, fromIn)
 		if _, err := c.NetworkingV1().Ingresses(m.Namespace).Create(ctx, in, metav1.CreateOptions{}); err != nil {
-			return err
+			if !oktetoErrors.IsAlreadyExists(err) {
```

Comment:
It could be a RC here is another `okteto deploy` command is diverting the same ingress in parallel. Since they both would create the same ingress yaml, it's safe to ignore this error

Reply:
I was going to propose the same thing. The function that we need to use is: https://github.com/kubernetes/apimachinery/blob/master/pkg/api/errors/errors.go#L159